src/scenes/Users/Detail/UserDetail.tsx

```diff
@@ -111,6 +118,24 @@ export const UserDetail = () => {
             loading={!user}
           />
           {user ? <EditUser user={user} {...editUserState} /> : null}
+
+          <DisplayProperty
+            label="Partners"
+            value={
```

Comment:
Idk if using `DisplayProperty` is the best thing here. It's more designed for simple values instead of a list of cards. 
For example, right now the loading state is completely different.

Reply:
Got it, updated!